R/create_episode_file.R

```diff
@@ -138,13 +138,13 @@ create_episode_file <- function(
     episode_file <- episode_file %>%
       dplyr::mutate(
         sc_send_lca = NA,
-        sc_living_alone = NA,
-        sc_support_from_unpaid_carer = NA,
-        sc_social_worker = NA,
-        sc_type_of_housing = NA,
-        sc_meals = NA,
-        sc_day_care = NA,
-        sc_latest_submission = NA,
+        # sc_living_alone = NA,
+        # sc_support_from_unpaid_carer = NA,
+        # sc_social_worker = NA,
+        # sc_type_of_housing = NA,
+        # sc_meals = NA,
+        # sc_day_care = NA,
+        # sc_latest_submission = NA,
```

Comment:
```suggestion
```

will remove this lanes as they will be added by join_sc_client_ep function

Reply:
Sounds good! The reason they are NA here is because they are 'declared' variables for when we do not have this available. e.g. year 23/24 currently does not have social care included but we have these variables set as NA for consistency. Do we have that option for the new `join_sc_client`? or should this just stay here for when we need to declare the variables? Happy to call to discuss!